src/main/resources/api/swagger-ui.html

```diff
@@ -0,0 +1,98 @@
+<!-- Copied from the swagger-ui static file -->
```

Comment:
What happens if a new version of the code generator shows up and changes this file?
Or asked another way, why is it safe to drop this file into the source?